test/common/validity_tests.rb

```diff
@@ -53,7 +53,7 @@ def test_validity_make_valid
           skip "#make_valid not handled by current implementation" unless implements_make_valid?
 
           assert_equal(bowtie_polygon_expected_area, bowtie_polygon.make_valid.area)
-          assert_raises(RGeo::Error::InvalidGeometry) do
+          assert_raises(RGeo::Error::UnsupportedOperation) do
```

Comment:
So to fix this for all do we need to check unsupported operation before validity?